### ISSUE TITLE
Install Android

### ISSUE THREAD

**Issue by z3t0** [Manouchehri/vi8#8]
Hi All.

@GalacticStryder

I currently have a Chuwi Vi8 with Windows 10 only (the android partition was formatted and deleted) and would like to install Android only. Does anyone know how I can go about doing this? Also, I dont have a USB keyboard/hub is there any way to do it without this?

Thanks!

**Comment by z3t0** [comment]
Also, Does anyone currently have the native dual boot configuration? If so can you please share some details in regards to the partition scheme.

**Comment by sigboe** [comment]
I think you can look at the image that is posted on needrom, and see if there are Android instructions there.

**Comment by GalaticStryder** [comment]
@z3t0 You can do it on Linux using fastboot commands when in DnX mode. Or use Windows: Intel has an app for that, you'll need to disable driver signature and then run the app to install the img. I'll link you the guide for that.
It's in spanish: https://mega.nz/#F!uEAzBQjT!S7jMaqeQSVC0-QlgCkVXGQ
Although very easy to understand, better than Chinese imo.

**Comment by sigboe** [comment]
@Manouchehri is the goal to support InsydeH2O's Android implementation or AOSP / Android-x86 ??

**Comment by Manouchehri** [comment]
@sigboe I don't _think_ building an AOSP image would be that hard since we've/you've figured out where most of the drivers are located. That being said, I haven't done it myself before.

Probably figuring out how to flash back the Chuwi/InsydeH2O stock image and then working on AOSP would be easier.

**Comment by GalaticStryder** [comment]
@Manouchehri Yeah, but building AOSP without a source tree first is a bit weird IMO. I don't know if they release their kernel sources (Chuwi), if not, we'll need to use the pre-compiled one.

**Comment by sigboe** [comment]
A big reason to do AOSP is that I think we can get a 64bit userspace...

**Comment by Manouchehri** [comment]
@GalaticStryder Intel releases a AOSP fork themselves. I think most/all of the sources are already online?

Anyone looked at the Android BSP yet? 

http://www.insyde.com/products/androidsolutions/android_portal

I found some public BSPs for M/D/I devices, but not the Bay Trail-T.

ftp://ftp.windriver.com/public/IAS/android_50md/
https://gist.github.com/shakalaca/f43be72ccd0ca041fb91

**Comment by Manouchehri** [comment]
Android Intel:

https://01.org/android-ia/

**Comment by GalaticStryder** [comment]
@sigboe You mean compiling AOSP with this makefile: https://github.com/android-ia/vendor_intel_baytrail/blob/master/minnow_max/minnow_64p.mk

**Comment by sigboe** [comment]
@GalaticStryder Wow thats a start... 

But I must say I do not have the competence nor the time to look into Android at this very instant. However, it seams we have 3 candidates here
1. InsydeH2O build (AOSP 4.4.4)
2. Android-ia (AOSP verson ???)
3. Repacking Chuwi shipped rom in a easier to flash manner. (AOSP 4.4.4)

The former and the later being two ways of doing the same thing really. While option 2 from intel is a cleaner solution that might take more work.

**Comment by Manouchehri** [comment]
@GalaticStryder That's for a Bay Trail-I and not the Bay Trail-T though.
